thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -3018,6 +3035,13 @@ class CLookupJoinActivityBase : public CInMemJoinBase<HTHELPER, IHThorHashJoinAr
                 activeStats.setStatistic(StNumSmartJoinDegradedToLocal, aggregateFailoversToLocal); // NB: is going to be same for all slaves.
             activeStats.setStatistic(StNumSmartJoinSlavesDegradedToStd, aggregateFailoversToStandard);
         }
+        if (overflowWriteFileIO)
```

Comment:
Does there need to be a critical block incase the overflow file, or the rhsSlaveRows are killed when this is being called?